pipeline/terraform/modules/es_index/main.tf

```diff
@@ -23,6 +23,7 @@ resource "elasticstack_elasticsearch_index" "the_index" {
   analysis_filter      = local.analysis.filter
   analysis_char_filter = local.analysis.char_filter
   analysis_tokenizer   = local.analysis.tokenizer
+  deletion_protection  = false
```

Comment:
Is it to do with the api key and secret? ie. deleting them when an old/unused pipeline is taken down?

Reply:
This is also a mistake.  I added it because I needed to destroy a pipeline that had gone funny, then forgot to remove it.

I'll raise a ticket about this, as I think this should be something we pass in from the top level to be very explicit about what we are doing.